examples/gameserverallocation.yaml

```diff
@@ -31,33 +31,32 @@ spec:
   # flexibility.
   # Below is an example of a GameServer allocated against a given fleet.
   # See: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for more details
-  required:
-    matchLabels:
-      game: my-game
-    matchExpressions:
-      - {key: tier, operator: In, values: [cache]}
-    # [Stage:Alpha]
-    # [FeatureFlag:StateAllocationFilter]
-    # Specifies which State is the filter to be used when attempting to retrieve a GameServer
-    # via Allocation. Defaults to "Ready". The only other option is "Allocated", which can be used in conjunction with
-    # label/annotation/player selectors to retrieve an already Allocated GameServer.
-    gameServerState: Ready
-    # [Stage:Alpha]
-    # [FeatureFlag:PlayerAllocationFilter]
-    # Provides a filter on minimum and maximum values for player capacity when retrieving a GameServer
-    # through Allocation. Defaults to no limits.
-    players:
-      minAvailable: 0
-      maxAvailable: 99
-  # ordered list of preferred allocations out of the `required` set.
+  # The selectors is an ordered list of allocations set.
   # If the first selector is not matched, the selection attempts the second selector, and so on.
   # This is useful for things like smoke testing of new game servers.
   # This also support `matchExpressions`
-  preferred:
+  selectors:
```

Comment:
🤔  should this have the deprecated fields, like the .md reference does? WDYT?

Reply:
I like the idea of having the `gameserverallocation-deprecated.yaml` 👍🏻  (sorry for the delay, totally missed this comment).

Reply:
Done.